src/Subtitles/RTS.cpp

```diff
@@ -653,9 +677,20 @@ bool CText::CreatePath()
             }
         }
     }
-
     SelectFont(g_hDC, hOldFont);
 
+    if (m_style.fontSize > MAXGDIFONTSIZE) {
+        double fact = m_style.fontSize / MAXGDIFONTSIZE;
+        for (ptrdiff_t i = 0; i < mPathPoints; i++) {
+            double x, y, z, xx, yy, zz;

```

Comment:
```
mpPathPoints[i].x *= fact;
mpPathPoints[i].y *= fact;
```

Reply:
Yeah that's better. I copied some other code and didn't simplify it.

Should put this in a function to keep it clean probably.